sqlglot/dialects/duckdb.py

```diff
@@ -321,6 +325,10 @@ class Generator(generator.Generator):
             exp.DataType.Type.UINT: "UINTEGER",
             exp.DataType.Type.VARBINARY: "BLOB",
             exp.DataType.Type.VARCHAR: "TEXT",
+            exp.DataType.Type.TIMESTAMP_S: "TIMESTAMP_S",
+            exp.DataType.Type.TIMESTAMP_MS: "TIMESTAMP_MS",
+            exp.DataType.Type.TIMESTAMP_US: "TIMESTAMP_US",
```

Comment:
We could potentially leave this out as parse target and just turn `TIMESTAMP_US` into `TIMESTAMP`, but I wasn't sure exactly how to do that.

Reply:
Done!